server/grpc_server.go

```diff
@@ -1,3 +1,13 @@
+/*
```

Comment:
lets move this comment to server.go and mention that there's two server implementations (socket and grpc)